src/features/DebugSession.ts

```diff
@@ -68,7 +68,8 @@ export class DebugSessionFeature implements IFeature, DebugConfigurationProvider
             const platformDetails = getPlatformDetails();
             const versionDetails = this.sessionManager.getPowerShellVersionDetails();
 
-            if (platformDetails.operatingSystem !== OperatingSystem.Windows) {
+            // Cross-platform attach to process was added in 6.2.0-preview.4
+            if (versionDetails.version < "6.2.0" && platformDetails.operatingSystem !== OperatingSystem.Windows) {
                 const msg = "Attaching to a PowerShell Host Process is supported only on Windows.";
```

Comment:
Seems like we should change this message to indicate it's is supported on Windows and PowerShell Core >= 6.2.0.

Reply:
wow duh. Thoughts on this?

```suggestion
                const msg = "Attaching to a PowerShell Host Process is supported on Windows and PowerShell Core 6.2 and up on Non-Windows.";
```

Reply:
How about this instead:
```
Attaching to a PowerShell Host Process on ${osName} requires PowerShell 6.2 or higher.
```